mujoco_warp/testspeed.py

```diff
@@ -146,10 +146,14 @@ def _main(argv: Sequence[str]):
     solver, cone = mjw.SolverType(m.opt.solver).name, mjw.ConeType(m.opt.cone).name
     integrator = mjw.IntegratorType(m.opt.integrator).name
     iterations, ls_iterations, ls_parallel = m.opt.iterations, m.opt.ls_iterations, m.opt.ls_parallel
+    if ls_parallel:
```

Comment:
nit: "steps" is a bit overloaded term, i would avoid using it in this context

there's plenty of options but perhaps I'd keep it simple and just use "iterations" whether it's the parallel or iterative linesearch